### PR TITLE
pieeprom-2022-04-14: Ethernet auto IDDQ fix & EEPROM update diagnostics

### PR DESCRIPTION
**Pull request by timg236**
* Fix netboot reboot failure on Pi 4B R1.1 if OS enables IDDQ power saving
     https://github.com/raspberrypi/rpi-eeprom/issues/417
* Fix incorrect error code (configuration error) on EEPROM update failure.
* Enable more verbose errors for EEPROM update failures

### CODE REVIEW

**Comment by timg236** [comment]
@peterharperuk

**Comment by peterharperuk** [comment]
Looks good

File: firmware/release-notes.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Raspberry Pi4 bootloader EEPROM release notes
 
+## 2022-04-22 - Add pieeprom-2022-04-22 release - BETA
+   * Fix netboot reboot failure on Pi 4B R1.1 if OS enables IDDQ power saving
+     https://github.com/raspberrypi/rpi-eeprom/issues/417
+   * Fix incorrect error code (configuration error) on EEPROM update failure.
```
Comment (lurch) [inline firmware/release-notes.md:6]: Does this mean there's a new error code which needs to be added to https://github.com/raspberrypi/documentation/blob/develop/documentation/asciidoc/computers/configuration/led_blink_warnings.adoc ?

Comment (timg236) [reply firmware/release-notes.md:6]: No, it will most likely emit SPI error or write protect error instead i.e. previous behaviour

**Pull request merged**